Data Streams - Integrator guide update (#1764)
smartcontractkit/documentation: src/features/data-streams/common/gettingStarted.mdx

Note: If you want to learn how to use the _[Streams Direct](/data-streams#streams-direct-using-the-data-engine-with-your-own-bot) implementation_ of Data Streams, refer to the [Fetch and decode reports via a REST API](/data-streams/tutorials/streams-direct-api) guide or the [Stream and decode reports via WebSocket](/data-streams/tutorials/streams-direct-ws) guide.

<DataStreams section="asideDisclaimer" />

## Before you begin

> 

Reviewer: There's also an Arbitrum Sepolia ETH faucet at Alchemy (https://www.alchemy.com/faucets/arbitrum-sepolia) if you want to include that as an option

## Deploy the Chainlink Automation upkeep contract

> Deploy an upkeep contract that is enabled to retrieve data from Data Streams. For this example, you will read from the ETH/USD Data Streams feed with ID `0x00027bbaff688c906a3e20a34fe951715d1018d262a5b66e38eda027a674cd1b` on Arbitrum Sepolia. See the [Data Streams Feed IDs](/data-streams/stream-ids) page for a complete list of available assets, IDs, and verifier proxy addresses.

Reviewer: ```suggestion
Deploy an upkeep contract that is enabled to retrieve data from Data Streams. For this example, you will read from the ETH/USD Data Streams feed 
 using the Arbitrum Sepolia feed ID `0x00027bbaff688c906a3e20a34fe951715d1018d262a5b66e38eda027a674cd1b`. See the [Data Streams Feed IDs](/data-streams/stream-ids) page for a complete list of available assets, IDs, and verifier proxy addresses.
```
